src/graphql/queries/personQuery.js

```diff
@@ -44,6 +44,21 @@ export default
         slug
       }
     }
+    projects {
```

Comment:
@lorenjohnson this is where I actually start to get concerned about performance, if every time we do a person query we load all this stuff. this is what I think is one of the things making Hylo slow as compared to loading data as needed when its needed. for example the User profile settings page is now very slow. so... we may someday want to go back to @wushenshen original implementation with separate queries, but for now this is fine :)

Reply:
The projects will only be queried when they are queried.